src/bridge/AbsBridge.sol

```diff
@@ -66,6 +70,18 @@ abstract contract AbsBridge is Initializable, DelegateCallAware, IBridge {
         }
         _;
     }
+    modifier whenNotPaused() {
```

Comment:
I think it would be nice to have more granular pauses available in the bridge. Eg:
* pause withdrawal / outbox
* pause inbox / delayed messages (we already have this in the inbox contract so maybe we could remove it there and put it here instead)
* pause seq inbox messages

Reply:
As also mentioned here https://github.com/OffchainLabs/nitro-contracts/pull/74#pullrequestreview-1709957909
we can use OZ AccessControl by having a BridgeStorage base contract that inherit the current storage layout so it doesn't get shifted downward.